unitTests/ui/ResultListTest.ts

```diff
@@ -784,11 +784,48 @@ export function ResultListTest() {
         describe('enableScrollToTop set to', () => {
           let infiniteScrollContainer: HTMLElement;
           let allResult: IQueryResults;
+          let scrollToSpy: jasmine.Spy;
+          let scrollTopSpy: jasmine.Spy;
+
           beforeEach(() => {
-            infiniteScrollContainer = { scrollTop: 500 } as HTMLElement;
+            document.createElement('div');
+            infiniteScrollContainer = document.createElement('div');
+            infiniteScrollContainer.style.height = '10000px';
+            infiniteScrollContainer.style.overflow = 'scroll';
+            document.body.appendChild(infiniteScrollContainer);
+            scrollToSpy = spyOn(infiniteScrollContainer, 'scrollTo');
+            let pValue = 500;
+            scrollTopSpy = jasmine.createSpy('scrollTop', v => (pValue = v));
+            Object.defineProperty(infiniteScrollContainer, 'scrollTop', {
+              get: () => pValue,
+              set: scrollTopSpy
+            });
             allResult = FakeResults.createFakeResults(50);
           });
 
+          afterEach(() => {
+            document.body.removeChild(infiniteScrollContainer);
+          });
+
+          it('should scroll to top on query success even in IE11', done => {
+            infiniteScrollContainer.scrollTo = undefined;
+            const option: IResultListOptions = {
+              enableInfiniteScroll: true,
+              infiniteScrollContainer: infiniteScrollContainer,
+              enableScrollToTop: true
+            };
+            test = Mock.basicComponentSetup<ResultList>(ResultList, option);
+            test.cmp.currentlyDisplayedResults.push(allResult.results[1]);
+            (test.cmp.queryController.fetchMore as jasmine.Spy).and.returnValue(Promise.resolve(allResult));
+            test.cmp.displayMoreResults(50).then(() => {
+              Simulate.query(test.env);
+              setTimeout(() => {
```

Comment:
I think it should be possible to use `jasmine.clock()` here, instead of timeout(s).

And if you would be kind enough to modify the other tests in this suite to use it also, that would be cool ;)

Reply:
If you don't mind that i make change of scope of this PR then no prob I'll do it :)

Reply:
Tests break too much will only do the change for this part.